### ISSUE TITLE
[FEATURE] Stats for nerds

### ISSUE THREAD

**Issue by Ultimatum22** [BrightDV/BoxBox#106]
**Is your feature request related to a problem? Please describe.**
Not the most useful feature but how about some (unneeded) stats. I sometimes look at [www.statsf1.com](https://www.statsf1.com) and this is fine for some (more) scraping.

For example I like these stats to begin with:
https://www.statsf1.com/en/statistiques/pilote/victoire/grille.aspx
https://www.statsf1.com/en/statistiques/pilote/meilleurtour/nombre.aspx

How it looks I have no clue yet and how to extend and what stats to use. I am curious about what you opinion is about such a feature.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**
Add any other context or **screenshots** about the feature request here.

**Comment by BrightDV** [comment]
That could be a nice idea, depending on how it is added. It was already requested in #10 but I wasn't focused on that topic.
Now I think it is the right time to add it.
However, the biggest issue that I see is that the website isn't too big and I don't want to force them to pay more bandwidth (there isn't even ads). 
That said, I've thinked about three places for the stats:
- In the driver/team details, with a tab "Stats"
- In the "Compare Screen" that I added some time ago (experimental features) wich does not work, but it could, using this website.
- In the circuit details screen, with maybe another tab.

I'm not sure whether adding a full section about stats which would only reproduce statsf1's website would be a good way to go.

Also, I've searched a bit and it seems that there is a lot of F1 stats websites, so the requests could be separated between the websites...

**Comment by Ultimatum22** [comment]
Yes fair point. I might combine it. I see that ergast has some useful stats, I'll see if I can make a small stats tab as poc.

**Comment by BrightDV** [comment]
> Yes fair point. I might combine it. I see that ergast has some useful stats, I'll see if I can make a small stats tab as poc.

For some f1 statistics-related websites, see: https://rothnic.github.io/formulapy/data_readme.html#data-sources
(Chicane F1 is very complete for drivers)

> I already used the caching manager, and was planning todo caching per url for about a week, the info hardly changes, so a week seems to be enough.

That's a good point. Also, I could make a proxy (like the boxbox server) if needed, so the cache would be centralized and would reduce (massively) the number of requests.

**Comment by Ultimatum22** [comment]
> For some f1 statistics-related websites, see: https://rothnic.github.io/formulapy/data_readme.html#data-sources (Chicane F1 is very complete for drivers)

I'll take a look at it. I'll make a small POC on the driver page.

> That's a good point. Also, I could make a proxy (like the boxbox server) if needed, so the cache would be centralized and would reduce (massively) the number of requests.

Yes that would be nice if you can do this.
